test/app-luatest/console_debugger_session_test.lua

```diff
@@ -22,11 +22,11 @@ local function tarantool_path(arg)
     return arg[index + 1]
```

Comment:
Yup, #7788 was planned followup. But still is not the topmost goal as we need to release vscode extension first.

Reply:
Strictly saying, you've committed to fix #7788 as soon as possible in order to push the first step to the trunk. It's definitely less critical than VScode extension, but more desired than `tdbg` or `tarantool -d`.

Reply:
With help of Nick Volynkin I was able to figure out which job was failing here (25 times) and why it has stopped to fail since some moment in the past. It was failing in release-asan workflow. Here is the latest log gathered http://multivac.hb.bizmrg.com/tarantool/tarantool/workflow_run_jobs/9206757219.log where we see the exactly the same problem which has been reported by https://github.com/tarantool/tarantool/issues/7882 and has been fixed then by https://github.com/tarantool/tarantool/pull/7885

This test is not flaky since then

Reply:
But still worth to highlight - refactoring of a test is under work, but is planned to be handled at a separate pull-request (to not block 3 currently available and ready pull request from being considered).